example-flask/app.py

```diff
@@ -4,7 +4,7 @@
 from flask import Flask
 from flask_cors import CORS
 
-from inigo_py import FlaskMiddleware
+from inigo_py.flask import Middleware
 from middleware import LogBlockedInigoRequestsMiddleware, LogMiddleware, AuthMiddleware
```

Comment:
basically this middleware doesn't work because InigoContext is not used

Reply:
`LogBlockedInigoRequestsMiddleware`

Reply:
Auth middleware without InigoContext used also doesn't make sense. Removed those.